v3/src/data-interactive/data-interactive-adornment-types.ts

```diff
@@ -23,7 +23,7 @@ export type DIStandardDeviationAdornmentValues = Partial<SnapshotIn<typeof Stand
 export type DIAdornmentValues = DICountAdornmentValues | DILsrlAdornmentValues | DIMeanAdornmentValues |
   DIMedianAdornmentValues | DIMovableValueAdornmentValues | DIStandardDeviationAdornmentValues
 
-const kAdornmentTypes = [kCountType, kLSRLType, kMeanType, kMedianType, kMovableValueType, kStandardDeviationType]
+const kAdornmentTypes = [kCountType, kLSRLType, kMeanType, kMedianType, kMovableValueType, kPercentType, kStandardDeviationType]
```

Comment:
The reason I added the type alias resolution system is so that the "Percent" adornment could be mapped to the `Count` adornment without having to pretend it exists elsewhere in the system. I'd rather leave this as a list of real adornments and change the `isAdornmentValues` function below. I've added a commit which addresses this.